crypto/src/error.rs

```diff
@@ -22,6 +22,9 @@ pub enum CryptoError {
     /// This error is emitted when the requested conversation couldn't be found in our store
     #[error("Couldn't find conversation")]
     ConversationNotFound(crate::prelude::ConversationId),
+    /// This error is emitted when the requested conversation already exists with the given if
+    #[error("Conversation already exists")]
+    ConversationAlreadyExists(crate::prelude::ConversationId),
```

Comment:
Maybe we should output the conversation id in the error name; What do you think? Maybe we should bring it up to @rohan-wire, as there's a concern around metadata leakage in possibly logs.

Same maybe with the `ClientNotFound` error, where the error stores the ClientId but doesn't output it in its `Display` impl

Reply:
When it comes to `ConversationId` & `ClientId` they are both present as path parameters in the backend API. So they are already public. I think it's fine